core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -91,6 +91,7 @@ private[spark] object JsonProtocol {
         executorRemovedToJson(executorRemoved)
       // These aren't used, but keeps compiler happy
       case SparkListenerExecutorMetricsUpdate(_, _) => JNothing
+      case SparkListenerBlockUpdate(_, _, _) => JNothing
```

Comment:
how often is this event posted, is it once per cached partition? If so there might be too many such events to log them.